sched/sched/sched_note.c

```diff
@@ -28,6 +28,9 @@
 #include <string.h>
 #include <assert.h>
 #include <errno.h>
+#ifdef CONFIG_SCHED_INSTRUMENTATION_HIRES
```

Comment:
could we remove #ifdef/#endif here? since line 37 don't use condition preprocess